src/System.Xml.XmlSerializer/pkg/System.Xml.XmlSerializer.pkgproj

```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Version>4.0.12.0</Version>
+    <Version>4.1.0.0</Version>
```

Comment:
Once we get the package version changes from master into dev/api this will actually just go away. We are no longer tying package versions with assembly versions. It will be 4.3.0 once we get that merge.